packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -873,7 +873,10 @@ export class Relayer {
 
       // Look up all blocks from contract deployment time to latest to ensure that a deposit, if it exists, is found.
       while (deposit === undefined) {
-        const fundsDepositedEvents = await this.l2Client.getFundsDepositedEvents(l2BlockSearchConfig);
```

Comment:
this call uses the new generalized L2client method.

Note that we should consider removing this event query block from this `Relayer.ts` file.